backbone.nativeview.js

```diff
@@ -19,10 +19,15 @@
   var ElementProto = (typeof Element !== 'undefined' && Element.prototype) || {};
 
   // Cross-browser event listener shims
-  var elementAddEventListener = ElementProto.addEventListener || function(eventName, listener) {
+  var elementAddEventListener = ElementProto.addEventListener ? function(eventName, listener, useCapture) {
+    return this.addEventListener(eventName, listener, useCapture);
```

Comment:
No need to check for useCapture. It's `false` by default, and we always pass `false` below. We could remove it entirely or just move it to here, I don't have a preference.